release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -64,9 +64,139 @@ The {product-title} installation program for Microsoft Azure now creates subnets
+* z/VM Emulated FBA devices on SCSI disks
+* 4K FCP block device
+
+These features are available only for {product-title} on IBM Z for {product-version}:
+
+* HyperPAV enabled on IBM Z /LinuxONE for the virtual machines for FICON attached ECKD storage
+
+[discrete]
+==== Restrictions
+
+Note the following restrictions for {product-title} on IBM Z and LinuxONE:
+
+* {product-title} for IBM Z does not include the following Technology Preview features:
+** Precision Time Protocol (PTP) hardware
+
+* The following {product-title} features are unsupported:
+** Automatic repair of damaged machines with machine health checking
+** CodeReady Containers (CRC)
+** Controlling overcommit and managing container density on nodes
+** CSI volume cloning
+** CSI volume snapshots
+** FIPS cryptography
+** Multus CNI plug-in
+** NVMe
+** OpenShift Metering
+** OpenShift Virtualization
+** Tang mode disk encryption during {product-title} deployment
+
+* Worker nodes must run {op-system-first}
+* Persistent shared storage must be provisioned by using either NFS or other supported storage protocols
+* Persistent non-shared storage must be provisioned using local storage, like iSCSI, FC, or using LSO with DASD, FCP, or EDEV/FBA
+
 [id="ocp-4-9-ibm-power"]
 === IBM Power Systems
 
+With this release, IBM Power Systems are now compatible with {product-title} {product-version}. For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on IBM Power Systems]
+* xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on IBM Power Systems in a restricted network]
+
+[discrete]
+==== Notable enhancements
+
+The following new features are supported on IBM Power Systems with {product-title} {product-version}:
+
+* Helm  
+* Support for Power10 
+* Support for multiple network interfaces
+* Service Binding Operator 
+
+[discrete]
+==== Supported features
+
+The following features are also supported on IBM Power Systems:
+
+* Currently, the following Operators are supported:
+** Cluster Logging Operator
+** NFD Operator
+** OpenShift Elasticsearch Operator
+** Local Storage Operator
+** SR-IOV Network Operator
+** Service Binding Operator
+
+* Multipathing
+* Persistent storage using iSCSI
+* Persistent storage using local volumes (Local Storage Operator)
+* Persistent storage using hostPath
+* Persistent storage using Fibre Channel
+* Persistent storage using Raw Block
+* OVN-Kubernetes
+* 4K Disk Support
+* NVMe
+* Encrypting data stored in etcd
+* Three-node cluster support
+* Multus SR-IOV
+
+[discrete]
+==== Restrictions
+
+Note the following restrictions for {product-title} on IBM Power Systems:
+
+* {product-title} for IBM Power Systems does not include the following Technology Preview features:
```

Comment:
If you make changes to the Z section, consider mirroring them here.

Reply:
* The following {product-title} Technology Preview features are unsupported: